src/parser/deathknight/frost/modules/runicpower/RunicPowerTracker.ts

```diff
@@ -75,10 +79,22 @@ class RunicPowerTracker extends ResourceTracker {
     this.mostRecentTickTime = event.timestamp;
   }
 
+  getHypothermicPresenceReduction(cost: number, timestamp: number) {
+    
+    if (this.selectedCombatant.hasBuff(SPELLS.HYPOTHERMIC_PRESENCE_TALENT.id, timestamp)) {
+      const newCost = cost * (1 - HYPOTHERMIC_PRESENCE_COST_REDUCTION);
+      this._totalHypothermicPresenceReduction += (cost - newCost);
+      cost = newCost;
+    }
+
+    return cost;
```

Comment:
wouldn't you just return newCost here? Not blocking for any merge just wondering.

Reply:
I just like having the original argument returned as is if the condition fails and also like having one return point if I'm returning a non bool